jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -34,6 +34,13 @@ class Catalogs(PluginTemplateMixin, ViewerSelectMixin, HasFileImportSelect, Tabl
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.close_in_tray`
     * :meth:`zoom_to_selected`
+    * :attr:`results_available`
+    * :attr:`number_of_results`
+    * :attr:`catalog`
+    * :attr:`max_sources`
+    * :meth:`search`
```

Comment:
I thought the ask was only to expose `.search()`. Are these others necessary?

Reply:
The plugin is called Catalog Search, and the ticket said to expose the catalog search API. It could go either way but i interpreted that as all public methods and attributes in the plugin.

Reply:
Maybe @kecnry can clarify since he opened the ticket...